database/factories/ModelFactory.php

```diff
@@ -452,5 +452,19 @@ function (Faker\Generator $faker, $attributes) {
             [
               'uuid' => $uuid
             ];
-    }
-);
+});
+
+// Delivery - a schedule of vouchers sent somewhere
+$factory->define(App\Delivery::class, function (Faker\Generator $faker, $attributes) {
+
+    $centre_id = (empty($attributes['centre_id']))
+        ? factory(App\Centre::class)->create()
+        : $attributes['centre_id']
+    ;
+
+    return [
+        'range' => '',
```

Comment:
Deliberately no default range?

Reply:
Rang is a text string. I have yet to impelment some kind of auto-range generator.